src/util/ValidationUtil.js

```diff
@@ -33,6 +33,17 @@ define(['okta'], function (Okta) {
     }
   };
 
+  // Validate that the password enroll matches for IdX. backwards compat, simplify later
+  fn.validatePasscodesMatch= function (model) {
```

Comment:
function is not used at all now. delete it?

Reply:
ah yeah, let me revert this file